### PR TITLE
Our picks

### PR DESCRIPTION
**Pull request by faraidoun**

### CODE REVIEW

File: sass/modules/item/_our-picks.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                margin-bottom: 0.75em;
+            }
+
+            h5{
+                opacity: 0.6;
+                margin-bottom: 2em;
+            }
+
+            .sub-item{
+                float: left;
```
Comment (Mekaeil) [inline sass/modules/item/_our-picks.scss]: please read cleaner code, notice the nesting code...

**Comment by faraidoun** [comment]
pleas see my changes

File: sass/modules/item/_our-picks.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+        h4{
+        text-align: center;
+        opacity: 0.6;
+        margin-top: 1em;
+        }
+    }//header of styles
+
+    .item{
+        float: left;
```
Comment (Mekaeil) [inline sass/modules/item/_our-picks.scss]: Write clean code!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

File: sass/modules/item/_our-picks.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        }
+    }
+
+
+    /*--------------------------------
+    ***  
+    **	 RESPONSIVE SECTIONs
+    */
+
+    @include lcd{
```
Comment (Mekaeil) [inline sass/modules/item/_our-picks.scss]: Transfer Responsive section to code...

File: sass/modules/item/_our-picks.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        border-right        : none;
+    }
+    }
+
+    .title{
+        opacity             : 0.5;
+        margin-bottom       : 0.5em;
+    }
+    }
+    }
```
Comment (Mekaeil) [inline sass/modules/item/_our-picks.scss]: Edit nesting and make it Cleaner!

**Pull request merged**